pkg/operator/k8sutil/service.go

```diff
@@ -42,6 +43,8 @@ func CreateOrUpdateService(
 		if err != nil {
 			return nil, fmt.Errorf("failed to update service %s. %+v", name, err)
 		}
+	} else {
+		logger.Debug("created service %s", s.Name)
```

Comment:
```suggestion
		logger.Debugf("created service %s", s.Name)
```

Reply:
You have highly trained eyes to see these logging issues!